client/gutenberg/extensions/publicize/store/actions.js

```diff
@@ -18,18 +23,19 @@ export function setConnections( postId, connections ) {
 }
 
 /**
- * Returns an action object used in signalling that
- * we're refreshing the Publicize connections for a post.
+ * Requests the Publicize connections for a post by its ID.
  *
  * @param {Number} postId      ID of the post.
  *
  * @return {Object} Action object.
  */
-export function refreshConnections( postId ) {
-	return {
-		type: 'REFRESH_CONNECTIONS',
-		postId,
-	};
+export function* refreshConnections( postId ) {
```

Comment:
Why not define this in `resolvers` instead, and do the renamed export here instead? We're exporting it from there anyway, and it's no longer really an action, so I think it'd fit better in `/resolvers`.

Reply:
I was on the fence about this one. I think in Gutenberg nomenclature, it still qualifies as an action (we're still using it as one after all), just one that happens to be a coroutine.

To me, what we're doing is better mapped by how it is right now -- having an action that then also happens to be used as an resolver (whose name needs to align with a selector), so I'd prefer things as they are. I am however not feeling super strongly about this.

Reply:
We can always follow-up later on that. 

:shipit: